pom.xml

```diff
@@ -879,6 +879,7 @@
         <module>modules/swagger-codegen-cli</module>
         <module>modules/swagger-codegen-maven-plugin</module>
         <module>modules/swagger-generator</module>
+        <module>modules/swagger-codegen-common-feign</module>
```

Comment:
Maybe `modules/languages/swagger-codegen-common-feign` ? So we could have other modules for other JVM languages/frameworks.